app/models/training_module.rb

```diff
@@ -55,15 +55,15 @@ def self.reload_module(slug:)
     # and can load slides for brand-new modules.
     TrainingLibrary.flush
     TrainingLibrary.load
-    TrainingModule.load(slug: [slug])
+    self.load(slug: slug)
```

Comment:
I'm testing this out, and I was wrong about being able to do this based on the slug only. It doesn't actually know the slugs from the wiki page that defines the available modules, because they are all json pages that don't embed the slug (unlike the index of slides, which do). So I think we'll need to load all the modules each time (but still, only the slides for the selected module).